openpype/hosts/maya/plugins/publish/validate_path_for_plugin.py

```diff
@@ -0,0 +1,51 @@
+import os
+
+from maya import cmds
+
+import pyblish.api
+
+from openpype.pipeline.publish import ValidateContentsOrder
+
+
+class ValidatePathForPlugin(pyblish.api.InstancePlugin):
+    """
+    Ensure Paths in Non-Maya Nodes(from plugins
+    such as Yeti, AbcExport) are correct
+    """
+
+    order = ValidateContentsOrder
+    hosts = ['maya']
+    families = ["workfile"]
+    label = "Non-existent Paths in Non-Maya Nodes"
```

Comment:
Seems like a mouthful. Maybe change to `Path Attributes( in Maya Plugin Nodes)`. (If we change the plugin name, we can skip the label in parentheses.)

Reply:
Relabeling to "Path attributes" I think is really good idea. If it's plug-ins only "Plug-in node path attributes".
"Existing paths in attributes" could also work.

Reply:
Nice `Plug-in Path Attributes`